terralab/auth_helper.py

```diff
@@ -159,7 +193,6 @@ def _exchange_code_for_response(
     code_key = "refresh_token" if grant_type == "refresh_token" else "code"
     data = {
         code_key: code,
-        "redirect_uri": redirect_uri,
```

Comment:
how come this is not needed anymore for the default login method?

Reply:
we didn't need it previously, but i didn't realize that then. everything still works fine without it (and it doesn't make sense that this step - exchanging code for access tokens - would care about the redirect_uri used to retrieve the code)